manual/manual/library/stdlib.etex

```diff
@@ -62,6 +62,7 @@ from being garbage-collected \\
 \subsubsection*{Arithmetic:}
 \begin{tabular}{lll}
 "Complex" & p.~\pageref{Complex} & Complex numbers \\
+"Float" & p.~\pageref{Float} & Floating-point numbers \\
```

Comment:
You missed the index part starting from line 100 for the html version of the manual, and line 147 for the latex version.

Reply:
Oops, thanks! Should be fixed now.

Reply:
The manual looks good now, thanks!

Reply:
Shouldn't it be explicitly said that it is _Double precision floating-point numbers_?

Reply:
Maybe. In http://caml.inria.fr/pub/docs/manual-ocaml/core.html#sec547, `type float` is described as being the type of "floating-point numbers" though.

Reply:
Sure but there is a chance that people find the module `Float` before the `type float`.